functions/controllers/user/getAUser.js

```diff
@@ -0,0 +1,39 @@
+const HttpStatus = require('http-status-codes');
+const { message, status } = require('../../util/constants');
+
+const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } = HttpStatus;
```

Comment:
`const { BAD_REQUEST, OK, INTERNAL_SERVER_ERROR } =  require('http-status-codes');`
this is more neat